src/AttributeTypeDictionnary.js

```diff
@@ -0,0 +1,64 @@
+//**************************************************************************************
+/**
+ * Map from RFC4519. Extract of AttributeType names with their corresponding OIDs.
+ */
+
+export const AttributeTypeDictionnary = {
```

Comment:
I used a Map to stored the default values and created the class with static methods like you suggested.
But to avoid confusion/mistake, I implemented a `getOID` method.

```js
AttributeTypeDictionary.set("1.2.3.4.5", "Some");
AttributeTypeDictionary.register([
  [ "6.7.8.9", "foo" ],
  [ "1.3.5.7.9", "bar" ]
]);

AttributeTypeDictionary.get("1.2.3.4.5"); // "Some"
AttributeTypeDictionary.get("1.2.3.4.5.6"); // "1.2.3.4.5.6"
AttributeTypeDictionary.get("Some"); // "Some"
AttributeTypeDictionary.getOID("Some"); // "1.2.3.4.5"
AttributeTypeDictionary.getOID("baz"); // null

```